app/src/main/java/com/mr/nemo/dragonfly/ui/entitiy/signup/SignUpScreenEffect.kt

```diff
@@ -0,0 +1,15 @@
+package com.mr.nemo.dragonfly.ui.entitiy.signup
+
+import com.mr.nemo.dragonfly.ui.entitiy.core.UiEffect
+import java.util.UUID
+
+sealed interface SignUpScreenEffect : UiEffect {
+
+    data class NavigateBack(
+        val id: String = UUID.randomUUID().toString()
+    ) : SignUpScreenEffect
+
+    data class NavigateToSignUpInfo(
+        val id: String = UUID.randomUUID().toString()
+    ) : SignUpScreenEffect
+}
```

Comment:
@bohdanzdaniekin maybe it's better to create general navigation effects instead of duplicating them for each screen?

Reply:
@slothdeboss Yeah, that makes sense, but I am going to do it later as soon as I`ll add navigation to the app